pkgs/top-level/aliases.nix

```diff
@@ -1253,8 +1253,6 @@ mapAliases {
 
   nixosTest = testers.nixosTest; # Added 2022-05-05
   nmap-unfree = throw "'nmap-unfree' has been renamed to/replaced by 'nmap'"; # Converted to throw 2024-10-17
-  nodejs-18_x = nodejs_18; # Added 2022-11-06
-  nodejs-slim-18_x = nodejs-slim_18; # Added 2022-11-06
```

Comment:
Keep those 2 alias, please add a `throw` for `nodejs_18`, `nodejs-slim_18` and `corepack_18`.